assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +359,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
+      return route.host.indexOf(defaultHost) !== 0;
```

Comment:
@liggitt Are we checking if the string starts with the defaultHost prefix or if it appears anywhere?